<launch>
  <!--param name="input_topic" value="/stereo_down/points2" /-->
  <param name="input_topic" value="/sense3d/scan/" />

  <arg name="scene" default="specification_scene.xml" />

  <node name="uwsim" pkg="uwsim" type="uwsim" args="--configfile $(find pm_grasp_visualization)/$(arg scene)" output="screen" respawn="false"/>
  <node name="grasp_server" pkg="pm_grasp_visualization" type="grasp_server" output="screen" respawn="false"/>
  <node name="waypoint_server" pkg="pm_grasp_visualization" type="waypoint_server" output="screen" respawn="false"/>
</launch>
